kotlin-grpc/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+dependencies {
+    implementation(project(":kotlin"))
+    implementation(project(":grpc"))
+    implementation(project(":junit5"))
+
+    // kotlin
+    implementation(libs.kotlin.coroutines.jdk8)
+    implementation(libs.kotlin.reflect)
+
+    testImplementation(libs.reactivestreams.tck)
+    testImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-test")
```

Comment:
Could you add this to `dependencies.toml` file and use the accesor?

Reply:
In 5c1c2ba, I added it to `dependencies.toml`.
Version catalog seems nice to managing various dependencies 👍